### PR TITLE
Settings and PageTitle commit

### PR DESCRIPTION
**Pull request by chengutman**
- Added UI for settingsScreen 
- Added Common Images with paths file to use in SettingsScreen and ChooseGameScreen
- Added PageTitle Component - will be the component that will display the Page title text : for example for Settings screen the text will be "Please choose a language" and for the choose game screen "Choose a game to start playing'

### CODE REVIEW

File: common/SettingsScreen.tsx [hunk 1]
```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import {LanguageId, LANGUAGES} from './localize';
-import {FlatListChooser} from './FlatListChooser';
```
Comment (avitalk1) [inline common/SettingsScreen.tsx:3]: Delete this

File: common/PageTitle.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  },
+  pageTitleText: {
+    fontSize: 24,
+    fontWeight: 'bold',
+  },
+});
+
+export function PageTitle() {
+  const route = useRoute();
+  const navigation = useNavigation();
```
Comment (avitalk1) [inline common/PageTitle.tsx]: Delete this

File: common/ChooseGameScreen.tsx [hunk 3]
```diff
@@ -1,29 +1,64 @@
 import React from 'react';
-import {FlatListChooser} from './FlatListChooser';
 import {getAllGameModules} from './gameModules';
 import {navigateNextFrame, useStoreContext} from './store';
 import {TitleBar} from './TitleBar';
 import {useNavigation} from '@react-navigation/native';
-import {commonStyles} from './common';
-import {View} from 'react-native';
+import {localize, LocalizeId} from './localize';
```
Comment (avitalk1) [inline common/ChooseGameScreen.tsx]: Delete only 'LocolizedId' from line six

**Comment by chengutman** [comment]
we will add a new PR

**Pull request closed without merging**